File: ios/MochiApp/App/MochiTab.swift
import Foundation

enum MochiTab: String, CaseIterable, Identifiable {
    case keyboard
    case fonts
    case create
    case themes
    case community

    var id: String { rawValue }

    var title: String {
        switch self {
        case .keyboard: return "Keyboard"
        case .fonts: return "Fonts"
        case .create: return "Create"
        case .themes: return "Themes"
        case .community: return "Community"
        }
    }

    var systemImage: String {
        switch self {
        case .keyboard: return "keyboard.fill"
        case .fonts: return "textformat"
        case .create: return "wand.and.stars"
        case .themes: return "paintpalette.fill"
        case .community: return "person.3.fill"
        }
    }
}

File: android/app/src/main/java/com/mochi/app/ui/MochiTab.kt
package com.mochi.app.ui

import androidx.compose.material.icons.Icons
import androidx.compose.material.icons.filled.AutoAwesome
import androidx.compose.material.icons.filled.Group
import androidx.compose.material.icons.filled.Keyboard
import androidx.compose.material.icons.filled.Palette
import androidx.compose.material.icons.filled.TextFields
import androidx.compose.ui.graphics.vector.ImageVector

/** Ported from ios/MochiApp/App/MochiTab.swift */
enum class MochiTab(val title: String, val icon: ImageVector) {
    KEYBOARD("Keyboard", Icons.Filled.Keyboard),
    FONTS("Fonts", Icons.Filled.TextFields),
    CREATE("Create", Icons.Filled.AutoAwesome),
    THEMES("Themes", Icons.Filled.Palette),
    COMMUNITY("Community", Icons.Filled.Group)
}
